__tests__/app.test.js

```diff
@@ -79,6 +92,57 @@ describe('/api',()=>{
                 })
             })
         })
+        test('GET:200 Should respond with array of article objects sort by created_at and default descending order',()=>{
+            return request(app).get('/api/articles').expect(200).then(({body})=>{
+                expect(body.articles).toBeSortedBy('created_at',{descending:true})
+            })
+        })
+        test('GET:200 Should respond with array of article objects with correct datatypes',()=>{
+            return request(app).get('/api/articles').expect(200).then(({body})=>{
+                body.articles.forEach((article)=>{
+                    expect(typeof article.author).toBe('string')
+                    expect(typeof article.title).toBe('string')
+                    expect(typeof article.article_id).toBe('number')
+                    expect(typeof article.topic).toBe('string')
+                    expect(typeof article.created_at).toBe('string')
+                    expect(typeof article.votes).toBe('number')
+                    expect(typeof article.article_img_url).toBe('string')
+                    expect(typeof article.comment_count).toBe('string')
+                    
```

Comment:
This test should be combined with your test on line 78 - no need to make it two separate blocks